### DISCUSSION TITLE
Slow Service Instance (ms) No Data

### DISCUSSION THREAD

**Discussion by BertOnline** [apache/skywalking#10086] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/17882970/205541937-8b86b328-b596-4d94-9176-b8e68aef0477.png)

After upgrading 9.3.0, I found that [Slow Service Instance] data is not displayed in some applications. Attached is the returned data.

[service_instance_resp_time.txt](https://github.com/apache/skywalking/files/10150392/service_instance_resp_time.txt)

**Comment by wu-sheng** [comment, marked as answer]
It seems working on the demo. And your attachments include the response. Could you run a local debug?

<img width="569" alt="image" src="https://user-images.githubusercontent.com/5441976/205545897-25f050e1-0528-4860-97a6-e4cf80747d6f.png">

**Reply by BertOnline** [reply]
OK，Thranks.
